.gitmodules

```diff
@@ -0,0 +1,7 @@
+[submodule "third_party/balpy"]
+	path = third_party/balpy
+	url = https://github.com/8ball030/balpy.git
+[submodule "third_party/multicaller"]
+	path = third_party/multicaller
+	url = https://github.com/8ball030/multicaller.git
```

Comment:
Why are these needed? It would be preferable to have PyPI released packages with version control

Reply:
These packages are dependencies used by the BabyDegen agent. While we could switch to using the PyPI packages, it would require 8baller to release the latest versions, as the most recent releases on PyPI are from 2023.

Reply:
UPDATE : 8baller has released updated versions on PyPI, I will integrate those in this merge, eliminating the need for the submodules.